domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/routes/RouteType.java

```diff
@@ -66,10 +65,13 @@ static public RouteType fromInt(final Integer fromValue) {
      * @param value the integer to associate with this enum values
      * @return true if the integer passed as parameter is expected for this enum, otherwise returns false
      */
-    static public boolean isEnumValueValid(final int value) {
+    static public boolean isEnumValueValid(final Integer value) {
+        if (value == null) {
+            return false;
+        }
         return Stream.of(RouteType.values())
                 .filter(enumItem -> enumItem.value == value)
                 .findAny()
                 .orElse(null) != null;
```

Comment:
Isn't `.orElse(null) != null` supposed to cover the case where  value is null?

Reply:
Indeed, the null check is redundant. I suggest in https://github.com/MobilityData/gtfs-validator/pull/423/commits/06018615c93a36e998a364d551fd01def54f3b76 to use `anyMatch` coupled with a try/catch block that will prevent NPE to be thrown when `isEnumValueMethod` arguent is `null`. This NPE is thrown by the filter `enumItem -> enumItem.value == value`.